src/main/java/dinkplugin/util/SerializedLoot.java

```diff
@@ -0,0 +1,11 @@
+package dinkplugin.util;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.Setter;
+
+@Data
+@Setter(AccessLevel.PRIVATE)
+public class SerializedLoot {
```

Comment:
Comment specifying where this comes from, if it's a struct from Runelite. maybe link to the file with a github link?